worker/backup_manifest.go

```diff
@@ -251,9 +247,21 @@ func ListBackupManifests(l string, creds *x.MinioCredentials) ([]*Manifest, erro
 		return nil, errors.Wrap(err, "error in listBackupManifests")
 	}
 
-	m, err := GetManifest(h, uri)
+	m, err := GetManifestForListbackup(h, uri)
 	if err != nil {
 		return nil, err
 	}
 	return m.Manifests, nil
 }
+
+func GetManifestForListbackup(h UriHandler, uri *url.URL) (*MasterManifest, error) {
```

Comment:
This is now how we should fix it. The code should follow a path that doesn't depend on what command is invoked. See if you can figure this out.